src/context/layers/impact.ts

```diff
@@ -28,6 +28,20 @@ export type ImpactLayerData = {
   impactFeatures: FeatureCollection;
 };
 
+const fetchApiData = async (url: string, apiData: any) =>
```

Comment:
Should we type `apiData`, here and on line 196?

Reply:
Yes ;-)

Reply:
I think we should also type the response here - we know what the data from the API will look like.